### DISCUSSION TITLE
Additional Filters for ZUI NW Graph

### DISCUSSION THREAD

**Discussion by GrizzlyAK** [zwave-js/zwave-js-ui#3643] [Ideas]
Using the NW Graph today I realized that there two filter options that I could have used, and wanted to drop them here for consideration.

Filters for:
(1) Number of Hops
(2) Data Rate

In my case, the graph is pretty "busy", and it morphs or moves around non-stop when viewing (I suspect this is due to the library in use). I was wishing for a way to show only the 3 or 4 hop Nodes to assess them independently, or maybe see the slower nodes isolated. It would simply help with viewing, assessing, and working with a smaller subset of nodes. I think it would add to the existing Location and Nodes filters.

**Comment by robertsLando** [comment]
Hi @GrizzlyAK, I can do that. In case of Data Rate I'm wondering if I should only show the low data rates or show the entire links between nodes and controller when at least one of them contains a low data rate?

**Reply by GrizzlyAK** [reply]
Not quite sure I'm visualizing what you mean. I was thinking basically a subset of what you show now, ie, only those meeting the filter for those containing low data rates. I think I'd prefer to see the entire chains from controller to all endpoints just each filtered chain, including the faster nodes, for perspective. But I'd be happy with showing only the slow paths too. 😁

**Reply by robertsLando** [reply]
> I think I'd prefer to see the entire chains from controller to all endpoints just each filtered chain, including the faster nodes, for perspective. But I'd be happy with showing only the slow paths too. 😁

Yeah this is what I meant :)
